securedrop_client/gui/widgets.py

```diff
@@ -128,33 +128,43 @@ def __init__(self) -> None:
         # Sync icon
         self.sync_icon = SyncIcon()
 
+        # Sync status bar with fixed width so that the left side of the
+        # activity status bar lines up with left pane
+        self.sync_status_bar = SyncStatusBar()
+        self.sync_status_bar.setFixedWidth(171)
+
         # Activity status bar
         self.activity_status_bar = ActivityStatusBar()
 
         # Error status bar
         self.error_status_bar = ErrorStatusBar()
 
-        # Create space the size of the status bar to keep the error status bar centered
-        spacer = QWidget()
+        # Create spacers the size of the sync icon and sync and activity status bars
+        # so that the error status bar is centered
+        sync_icon_spacer = QWidget()
+        sync_icon_spacer.setFixedWidth(42)
+
+        sync_status_bar_spacer = QWidget()
+        sync_status_bar_spacer.setFixedWidth(171)
 
-        # Create space ths size of the sync icon to keep the error status bar centered
-        spacer2 = QWidget()
-        spacer2.setFixedWidth(42)
+        activity_status_bar_spacer = QWidget()
```

Comment:
Why does this spacer not have a fixed size? (:thought_balloon: Because it didn't use to?)

Reply:
Yes that right, I've renamed `spacer` to `activity_status_bar_spacer` to make it a bit more clear what it corresponds to. It still has variable width because, like the activity status bar, its added to the layout with a stretch factor of 1.
`sync_status_bar` needed to be of fixed width so that the activity status bar would appear directly above the source list, so I gave it and `sync_status_bar_spacer` a fixed width.